scripts/delete_trailing_whitespace.sh

```diff
@@ -4,6 +4,8 @@
 # will be used (one up from the scripts directory where this script is located)
 REPO_DIR=${1:-"$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../"}
 
+echo "this script is deprecated, please use 'moose-format.sh' instead"
```

Comment:
Don't we still need this script for `*.py`, `*.i`, and `*.tex` files?
And it seems like `moose-format.sh` only does checking...